helm/superset/values.yaml

```diff
@@ -21,15 +21,19 @@
 
 replicaCount: 1
 
-## These requirements are used to build a requirements file which is then applied on init
-## of superset containers
-additionalRequirements:
-  - "psycopg2==2.8.5"
-  - "redis==3.2.1"
-
-## These apt packages are applied on init of superset containers
-additionalAptPackages: {}
-  # - nano
+# User ID directive
+# Note that this user must have appropriate permissions to run the bootstrap script
+runAsUser: 1000
+
+# Install additional packages and do any other bootstrap configuration in this script
+# For production clusters it's recommended to build own image with this step done in CI
+bootstrapScript: |
+  #!/bin/bash
```

Comment:
Did we want to comment all of these lines? Typo?

Reply:
Yes. I switched default uid to 1000 which will cause these lines to fail if those lines are uncommented. The purpose is to show users that this is a place to perform such tasks, but also that they must make their environment less secure by setting runAsUser value to 0.

Reply:
Understood. Can you please put it back to 0 and enable that script? Maybe just add some verbage indicating that this isn't something one should do in production? There's several folks that rely on that thing currently (including me 😀).

Reply:
Ok, this is now done.